R/data.R

```diff
@@ -65,3 +65,12 @@
 #' Derived from the `adsl` and `vs` datasets using `{admiral}` (\url{https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_advs.R})
 #'
 "admiral_advs"
+
+#' Pharmacokinetics Parameters Analysis Dataset
+#'
+#' An example vital signs analysis dataset
+#'
+#' @source
+#' Derived from the [adsl] and [pp] datasets using `{admiral}` (\url{https://github.com/Roche-GSK/admiral/blob/main/inst/templates/ad_adpp.R})
+#'
+"adpp"
```

Comment:
GIven our new naming convention this should be `admiral_adpp`.

Reply:
ok but I still have an issue here when I try to generate the doc